### PR TITLE
Carrier Name change and Length & Weight unit issue fixed

### PR DESCRIPTION
**Pull request by prantikdasgupta**
Fixes done for the following issues:
- Added the hookupdateCarrier to update the carrier id in our ps_sellstrom_carrier table.
- Passing the length unit and weight unit from the module to our server so that it supports KG & CM

### CODE REVIEW

File: views/templates/admin/hookheader.tpl [hunk 1]
```diff
@@ -109,11 +109,12 @@
 		$('#ajaxLoaderImgRemove').hide();
 
 		var existingContent = $('#form .delivery_options').html();
-		var dhlPattern = /Sellstrom\s+\-\s+DHL\s+[\w\|\-]+\s+/gi;
-		var dhlModifiedContent = existingContent.replace(dhlPattern, 'DHL ');
-		var replacePattern = /Sellstrom\s+\-\s+\w+\s+[\w\|\-]+\s+/gi;
-		var realContent = dhlModifiedContent.replace(replacePattern, '');
-		$('#form .delivery_options').html(realContent);
+		$('#form .delivery_options').html(existingContent);
+		//var dhlPattern = /Sellstrom\s+\-\s+DHL\s+[\w\|\-]+\s+/gi;
```
Comment (gaillafr) [inline views/templates/admin/hookheader.tpl]: Could you please directly remove these lines?

File: upgrade/install-0.1.6.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+*  @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*/
+
+if (!defined('_PS_VERSION_'))
+	exit;
+
+function upgrade_module_0_1_6($object)
+{
+	return $object->registerHook('updateCarrier');
```
Comment (gaillafr) [inline upgrade/install-0.1.6.php]: Why don't you add the others hooks?
$this->registerHook('backofficeHeader')
$this->registerHook('displayOrderDetail')

**Comment by gaillafr** [comment]
Thank you @prantikdasgupta, you PR has been merged manually: https://github.com/PrestaShop/sellstrom/commit/af7e66e50e5239aa8028f97b4dd736a3b3b64514

**Comment by prantikdasgupta** [comment]
Hello @gaillafr, I don't see my changes being merged to the main master. It is still showing 0.1.5 as the latest version. Could you please merge it? Our customers need the latest changes ASAP.

Thanks,
Prantik

**Pull request closed without merging**